bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/api/Websocket.java

```diff
@@ -402,8 +431,9 @@ private void onClosedSession(@Nullable Throwable throwable) {
         state = WebsocketState.DISCONNECTED;
         pingPongMap.clear();
         if (throwable != null) {
-            accountHandler.updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
-                    "@text/mercedesme.account.status.websocket-failure [\"" + throwable.getMessage() + "\"]");
+            logger.info("Websocket onClosedSession exception: {} - try to resume login", throwable.getMessage());
```

Comment:
This seems a bit technical. If the user needs to be made aware of this, perhaps the Thing status description could be set? Otherwise log level could be set to `debug`.

Reply:
Agree, changed it